ghost/core/core/shared/labs.js

```diff
@@ -15,7 +15,8 @@ const messages = {
 
 // flags in this list always return `true`, allows quick global enable prior to full flag removal
 const GA_FEATURES = [
-    'newsletterPaywall'
+    'newsletterPaywall',
+    'comments'
```

Comment:
We don't need this if you've removed all the checks for the flag btw, which it _looks_ like you have?

I would suggest either only making this change, or doing a thorough search for all flag usage and removing all of it, without adding this to the `GA_FEATURES` list.

Reply:
You are right, was a bit scared to remove it here 😅 Since we have a monorepo now, it is so much easier to remove it everywhere at once

Reply:
Swedish let's do it!!